src/Components/Card.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
```

Comment:
`import React, {useState} from 'react;`
React should be in scope when using JSX